src/Alert/index.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+import * as React from "react";
+import * as tokens from "@kiwicom/orbit-design-tokens";
+
+import * as Icons from "../icons";
+import { Button } from "../index";
+
+const Icon = ({ type, Icon: CustomIcon, customColor }) => {
+  switch (type) {
+    case "info":
+      return (
+        (CustomIcon && <CustomIcon customColor={customColor} />) || (
+          <Icons.InformationCircle customColor={customColor} />
+        )
+      );
+    case "success":
+      return (
+        (CustomIcon && <CustomIcon customColor={customColor} />) || (
+          <Icons.Check customColor={customColor} />
+        )
+      );
+    case "warning":
+      return (
+        (CustomIcon && <CustomIcon customColor={customColor} />) || (
+          <Icons.Alert customColor={customColor} />
+        )
+      );
+    case "critical":
+      return (
+        (CustomIcon && <CustomIcon customColor={customColor} />) || (
+          <Icons.AlertCircle customColor={customColor} />
+        )
+      );
+    default:
+      return null;
+  }
+};
+
+const backgroundAlert = {
+  info: tokens.backgroundAlertInfo,
+  success: tokens.backgroundAlertSuccess,
+  warning: tokens.backgroundAlertWarning,
+  critical: tokens.backgroundAlertCritical,
+};
+
+const colorTextAlert = {
+  info: tokens.colorTextAlertInfo,
+  success: tokens.colorTextAlertSuccess,
+  warning: tokens.colorTextAlertWarning,
+  critical: tokens.colorTextAlertCritical,
+};
+
+const colorIconAlert = {
+  info: tokens.colorAlertIconInfo,
+  success: tokens.colorAlertIconSuccess,
+  warning: tokens.colorAlertIconWarning,
+  critical: tokens.colorAlertIconCritical,
+};
+
+type Props = {
+  type: $Keys<typeof backgroundAlert | typeof colorTextAlert>,
+  title?: string,
+  Icon?: React.ComponentType<*>,
```

Comment:
Is Icon component receiving any props in this use case ? If no, please rather use React.Node